yarn.lock

```diff
@@ -5177,6 +5177,13 @@ getpass@^0.1.1:
   dependencies:
     assert-plus "^1.0.0"
 
+glob-parent@6.0.2:
+  version "6.0.2"
+  resolved "https://registry.yarnpkg.com/glob-parent/-/glob-parent-6.0.2.tgz#6d237d99083950c79290f24c7642a3de9a28f9e3"
+  integrity sha512-XxwI8EOhVQgWp6iDL+3b0r86f4d6AX6zSU55HfB4ydCEuXLXc5FcYeOu+nnGftS4TEju/11rt4KJPTMgbfmv4A==
+  dependencies:
+    is-glob "^4.0.3"
+
 glob-parent@^3.1.0:
```

Comment:
I believe the vulnerability will persist as we're still bringing in a vulnerable version here related to `chokidar` (i.e.  < `5.1.2.` . I believe we need to id the sub-dependency bringing it in and see if we can update to a non-vulnerable version.